### PR TITLE
MODUSERBL-180 Module needs to respond 404 when 404 is received from /authn/login and /token endpoints

### PR DESCRIPTION
**Pull request by pavankumar181**
## Purpose
The purpose of this PR is to add a 404 response for /auth/login and /token endpoints

## Approach
Added 404 response in AuthTokenClientImpl class for /token endpoint and for /auth/login they are already handling 404 response.

### CODE REVIEW

**Review by gurleenkaurbp — APPROVED** [review]

File: src/main/java/org/folio/rest/util/ExceptionHelper.java [hunk 1]
```diff
@@ -44,6 +45,11 @@ public static Response handleException(Throwable throwable) {
         .type(MediaType.TEXT_PLAIN)
         .entity(throwable.getMessage())
         .build();
+    } else if(throwable.getClass() == TokenNotFoundException.class){
```
Comment (steveellis) [inline src/main/java/org/folio/rest/util/ExceptionHelper.java:48]: Need space after `if`

**Review by steveellis — APPROVED** [review]
Please fix the small nits to fix before merging

**Review by Vignesh-kalyanasundaram — APPROVED** [review]

File: src/main/java/org/folio/rest/util/ExceptionHelper.java (hunk 1 above)
Comment (pavankumar181) [reply src/main/java/org/folio/rest/util/ExceptionHelper.java]: Added space

File: src/main/java/org/folio/rest/exception/TokenNotFoundException.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package org.folio.rest.exception;
+
+public class TokenNotFoundException extends RuntimeException {
+
+    public TokenNotFoundException() {
+        super();
+    }
```
[the remark this answers is not in the document]
Comment (pavankumar181) [reply src/main/java/org/folio/rest/exception/TokenNotFoundException.java]: removed

**Pull request merged**